README.md

```diff
@@ -110,32 +109,26 @@ This always compiles the latest nightly builds. If you want to build stable, do
 $ git checkout stable
 ```
 
-### 3.3 Simple One-Line Installer for Mac and Linux <a id="chapter-0033"></a>
-
-```bash
-bash <(curl https://get.parity.io -L)
-```
-
-### 3.4 Starting Parity Ethereum <a id="chapter-0034"></a>
+### 3.3 Starting Open Ethereum <a id="chapter-0034"></a>
 
 #### Manually
 
-To start Parity Ethereum manually, just run
+To start Open Ethereum manually, just run
 
 ```bash
 $ ./target/release/parity
 ```
 
-so Parity Ethereum begins syncing the Ethereum blockchain.
+so Open Ethereum begins syncing the Ethereum blockchain.
 
 #### Using `systemd` service file
 
-To start Parity Ethereum as a regular user using `systemd` init:
+To start Open Ethereum as a regular user using `systemd` init:
 
 1. Copy `./scripts/parity.service` to your
 `systemd` user directory (usually `~/.config/systemd/user`).
 2. Copy release to bin folder, write `sudo install ./target/release/parity /usr/bin/parity`
-3. To configure Parity Ethereum, write a `/etc/parity/config.toml` config file, see [Configuring Parity Ethereum](https://paritytech.github.io/wiki/Configuring-Parity) for details.
+3. To configure Open Ethereum, write a `/etc/parity/config.toml` config file, see [Configuring Parity Ethereum](https://paritytech.github.io/wiki/Configuring-Parity) for details.
```

Comment:
I guess we will eventually move that documentation to OpenEthereum, maybe say `old configuration` or something until then